app/assets/javascripts/admin/views/statistic/project_progress_report_view.js

```diff
@@ -1,42 +1,100 @@
 // @flow
 import * as React from "react";
-import { Spin, Table, Card } from "antd";
+import { Icon, Spin, Table, Card } from "antd";
 import Utils from "libs/utils";
+import FormatUtils from "libs/format_utils";
 import { getProjectProgressReport } from "admin/admin_rest_api";
 import type { APIProjectProgressReportType } from "admin/api_flow_types";
 import TeamSelectionForm from "./team_selection_form";
 
-const { Column } = Table;
+const { Column, ColumnGroup } = Table;
+
+const RELOAD_INTERVAL = 10 * 60 * 1000; // 10 min
 
 type State = {
+  settingsVisible: boolean,
```

Comment:
boolean variables should be name to be easily recognizable: isBar, areFooVisible, shouldXYZ